guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -89,6 +89,8 @@ ifdef::satellite[]
 include::topics/removing_satellite_tools_repository.adoc[leveloffset=+2]
 endif::[]
 
+include::common/modules/proc_migrating-ansible-content.adoc[leveloffset=+2]
```

Comment:
Nearly all of this applies to Satellite/RHEL customers, so I am thinking it should be walled off to Satellite only.